first homework/task2.js

```diff
@@ -1,21 +1,20 @@
 import csv from 'csvtojson';
-import { writeFile } from 'fs';
+import { createWriteStream } from 'fs';
 const csvFilePath='csv\\nodejs-hw1-ex1.csv';
 
-var data='';
+let stream=createWriteStream('test.txt');
 csv()
 .fromFile(csvFilePath)
-.on('error',(err)=>{
-	console.log(err)
-})
-.then((jsonObj)=>{
-	jsonObj.forEach((line)=>{
-                data+=JSON.stringify(line)+'\n';
-        })
+.subscribe((json,lineNumber)=>{
+                let outputLine={};
+                for(let key in json){
+                        outputLine[firstLettertoLower(key)]=json[key]
+                }
+                stream.write(JSON.stringify(outputLine)+'\n');
```

Comment:
use template string here instead of concatenation
https://www.w3schools.com/js/js_string_templates.asp